src/types/index.ts

```diff
@@ -153,6 +153,7 @@ export interface FormSettings {
   consent?: boolean;
   consentText?: string;
   logic?: FormLogic[];
+  multiselect: boolean;
```

Comment:
@afolasope No, this is incorrect. The `multiselect` should not be here. Why do we need a `multiselect` flag in the first place? A question already has a `type`.